bucket/video-thumbnails-maker.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "14.0.0.0",
+    "description": "Create video thumbnails.",
+    "homepage": "http://www.suu-design.com/projects_vtm.html",
+    "license": "Proprietary",
+    "url": "http://www.suu-design.com/Files/VTM/1414881427/VideoThumbnailsMaker_Setup.zip",
+    "hash": "cee9e8b4016b0f009a8e9b4452b61823f982d275315c96dbf9cb1a4a0e1b2303",
+    "pre_install": [
+        "Expand-7zipArchive -Path \"$dir\\VideoThumbnailsMaker_Setup.exe\" -DestinationPath \"$dir\"",
+        "Remove-Item \"$dir\\VideoThumbnailsMaker_Setup.exe\", \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall.exe\" -Force -Recurse"
+    ],
+    "bin": "VideoThumbnailsMaker.exe",
+    "shortcuts": [
+        [
+            "VideoThumbnailsMaker.exe",
+            "Video Thumbnails Maker"
+        ]
+    ],
+    "checkver": {
+        "url": "http://www.suu-design.com/downloads.html",
+        "regex": "(?sm)Video Thumbnails Maker</strong>.+v([\\d.]+).+/(?<random>\\d+)/VideoThumbnailsMaker_Setup\\.zip"
+    },
+    "autoupdate": {
+        "url": "http://www.suu-design.com/Files/VTM/$matchRandom/VideoThumbnailsMaker_Setup.zip"
```

Comment:
Why do you use `$matchRandom`?

> The number in the download URL (Files/VTM/1414881427/VideoThumbnailsMaker_Setup.zip) is constant (remains the same)

Reply:
It's a bad naming. I renamed it to `$matchFileid` (instead of `$matchRandom`) to avoid misunderstanding.

Reply:
Why are you using variable when it is always same??

From PR description:
> The number in the download URL (Files/VTM/1414881427/VideoThumbnailsMaker_Setup.zip) is constant (remains the same)

Reply:
> Why are you using variable when it is always same??

The number (Files/VTM/**1414881427**/VideoThumbnailsMaker_Setup.zip) changes every time when the file is updated. (for example, updating from version `14.0.0.0` -> `14.0.0.1`)

Reply:
Then why are you saying it is constant?

Reply:
What I mean is, the number will not change when you try to **download it again after a few hours**.  Maybe I shouldn't use the word "constant" in this situation?